web/src/main/java/org/akaza/openclinica/control/core/SecureController.java

```diff
@@ -704,28 +704,24 @@ else if (currentRole.getId() > 0
             // Set if enrollment is capped. Used by navBar.jsp to hide "Add Participant" link in the menu
             processRequest();
         } catch (InconsistentStateException ise) {
-            ise.printStackTrace();
-            logger.warn("InconsistentStateException: org.akaza.openclinica.control.SecureController: " + ise.getMessage());
-
+            logger.warn("InconsistentStateException: org.akaza.openclinica.control.SecureController: {}" , ise.getMessage(), ise);
             addPageMessage(ise.getOpenClinicaMessage());
             forwardPage(ise.getGoTo());
         } catch (InsufficientPermissionException ipe) {
-            ipe.printStackTrace();
-            logger.warn("InsufficientPermissionException: org.akaza.openclinica.control.SecureController: " + ipe.getMessage());
-
+            logger.warn("InsufficientPermissionException: org.akaza.openclinica.control.SecureController: {}" , ipe.getMessage(),ipe);
             // addPageMessage(ipe.getOpenClinicaMessage());
             forwardPage(ipe.getGoTo());
         } catch (OutOfMemoryError ome) {
-            ome.printStackTrace();
+            logger.error("Memmory full in the process: ",ome);
             long heapSize = Runtime.getRuntime().totalMemory();
             session.setAttribute("ome", "yes");
         } catch (Exception e) {
-            e.printStackTrace();
+            logger.error("Process is throwing exception: ",e);
             logger.error(SecureController.getStackTrace(e));
```

Comment:
second logger is no longer needed

Reply:
I have changed according to your comments